target/scripts/helpers/utils.sh

```diff
@@ -11,3 +11,8 @@ function _is_comment
 {
   grep -q -E "^\s*$|^\s*#" <<< "${1}"
 }
+
+function _log_date
+{
+  date +"%Y-%m-%d %H:%M:%S"
+}
```

Comment:
Outdated branch? This function should be in `logs.sh`

Reply:
https://github.com/docker-mailserver/docker-mailserver/pull/2498#issuecomment-1075040739

> I'm waiting on #2499 to get merged because this PR holds the `_log_with_date` function, which I want to use in `check-for-changes.sh` as well.